shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -76,6 +76,35 @@ typedef NS_ENUM(NSUInteger, FlutterTextAffinity) {
   return flutter::TextRange([base unsignedLongValue], [extent unsignedLongValue]);
 }
 
+@interface NSEvent (KeyEquivalentMarker)
+
+/// Internally marks that the event was received through performKeyEquivalent:.
+/// When text editing is active, keyboard events that have modifier keys pressed
+/// are received through performKeyEquivalent: instead of keyDown:. If such event
+/// is passed to TextInputContext but doesn't result in a text editing action it
+/// needs to be forwarded by FlutterKeyboardManager to the next responder.
```

Comment:
```suggestion
// Internally marks that the event was received through performKeyEquivalent:.
// When text editing is active, keyboard events that have modifier keys pressed
// are received through performKeyEquivalent: instead of keyDown:. If such event
// is passed to TextInputContext but doesn't result in a text editing action it
// needs to be forwarded by FlutterKeyboardManager to the next responder.
```